src/main/ml-modules/root/com.marklogic.smart-mastering/survivorship/merging/base.xqy

```diff
@@ -1087,6 +1092,46 @@ declare function merge-impl:build-final-headers(
   )
 };
 
+(:~
+ : Build a sequence of triples
```

Comment:
Please add a comment that says why namespace/function/at are attributes and not listed separately under `algorithms` (`algorithms` config allows reuse across multiple properties, which doesn't apply to triples; less likely to have a reference error).